frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './styles.module.scss';
+
+type Properties = {
+  style?: 'yellow' | 'black';
+  children: React.ReactNode;
+};
+
+const PageLayout: React.FC<Properties> = ({
+  children,
+  style = 'yellow',
```

Comment:
As I understand this prop is only for changing the bg-color -- if it's needed can't it be accomplished by custom outer `className` prop? because currently this `style` prop looks strange (first thought may be about style attr)

Reply:
done